tests/data/various_modules_package/__init__.py

```diff
@@ -9,10 +9,12 @@
 from ._reexport_module_4 import _two_times_reexported as two_times_reexported
 from .enum_module import _ReexportedEmptyEnum
 from file_creation._module_3 import Reexported
+from .class_module import ClassModuleClassD as ClMCD
```

Comment:
Since you import, and thus, reexport the declaration under an alias, the declaration also cannot be called `ClassModuleClassD` in the stubs. **It must be renamed to the alias.**

Reply:
The question should always be is

```py
from <@PythonModule or package name> import <@PythonName or declaration name>
```

legal in Python.